js/main.css

```diff
@@ -1,19 +1,38 @@
 body 
 {
 	margin: 0px;
-	min-width: 411.5px;
+	min-width: 700px;
 	overflow-x: hidden;
 	font: 12px 'Noto Sans' ,'lucida grande', helvetica, arial, sans-serif;
 }
 
 #item 
 {
-	width: 167.5px;
 	clear: both;
 	cursor: pointer;
 	background-color:#424242;
 	padding-top: 3px;
 	padding-bottom: 3px;
+	flex: 1;
+}
+
+#item1
```

Comment:
Use same class for all the 3 items, instead of using 3 ids

Reply:
but using same class we can't add 3 column's

Reply:
@akshatnitd Please response

Reply:
@akshatnitd please rest asap.

Reply:
@akshatnitd please look at this asap.

Reply:
Classes to each item isn't required.
Use - 
```

Reply:
@akshatnitd That give the previous shown result not in group of 3.